mlflow/utils/validation.py

```diff
@@ -81,6 +111,13 @@ def _validate_tag_name(name):
                               INVALID_PARAMETER_VALUE)
 
 
+def _validate_length_limit(entity_name, limit, value):
+    if len(value) > limit:
+        raise MlflowException(
+            "%s %s had length %s, which exceeded length limit of %s" %
```

Comment:
For clarity does it make sense to put the actual think that is being validated in quotes?

```
        raise MlflowException(
            "%s '%s' had length %s, which exceeded length limit of %s" %
            (entity_name, value, len(value), limit))

```